e2e-tests/spring-rest-openapi-v2/src/main/java/com/foo/rest/examples/spring/authenticatedswaggeraccess/AuthenticatedSwaggerAccessRest.java

```diff
@@ -0,0 +1,33 @@
+package com.foo.rest.examples.spring.authenticatedswaggeraccess;
+
+import com.foo.rest.examples.spring.security.accesscontrol.deleteput.ACDeletePutDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+@RestController
+@RequestMapping(path = "/api")
+public class AuthenticatedSwaggerAccessRest {
+
+
+
+
+    @GetMapping(value = "/endpoint1")
+    public ResponseEntity getResourceEnd1(@PathVariable("x") String x) {
+
+        return new ResponseEntity<>(x, HttpStatus.OK);
+
+    }
+
+    @GetMapping(value = "/endpoint2")
+    public ResponseEntity getResourceEnd2(@PathVariable("x") String x) {
```

Comment:
there is no `{x}` variable in the path?

Reply:
Hi Andrea,

Thanks for your comment. I have fixed this one.